adapters/juniper_srx/juniper_srx_configuration.php

```diff
@@ -408,10 +409,11 @@ function update_license()
     $ipaddr = $sms_sd_ctx->getIpAddress();
     $login = $sms_sd_ctx->getLogin();
     $passwd = $sms_sd_ctx->getPassword();
+    $sd_mgt_port = $sd->SD_MANAGEMENT_PORT;
```

Comment:
`$sd` is not initialized

Reply:
fixed by this commit https://github.com/openmsa/Adapters/pull/295/commits/d5ee499d2ebe0b444954df206cd7458cbd28e3e1

Reply:
line 416 should not be there

Reply:
fixed by commit  [4c21b11](https://github.com/openmsa/Adapters/pull/295/commits/4c21b11278e410b25f66da4800d8b072047a4202)